README.md

```diff
@@ -18,9 +18,12 @@ Data is transformed from these standard formats and loaded into Elasticsearch (t
     - **-t, --transformer** *Default: transformers/[[data file basename]].js* The transformer to use. This converts state-specific data formats to our [Point Data Schema](https://github.com/cfpb/grasshopper/blob/master/docs/point_data_spec.md)
     - **-h, --host** *Default: localhost* The elasticsearch host
     - **-p, --port** *Default: 9200* The elasticsearch port
+    - **-l, --log** *Default: debug* The elasticsearch log level
     - **--index** *Default: address* The elasticsearch index
     - **--type** *Default: point* The elasticsearch type
     - **--profile** *Default: default* The aws credentials profile in ~/.aws/credentials. AWS keys as environment variables will override this setting.')
+    - **--s_srs** *Auto-detects if possible* Source spatial reference system. Needed for untransformed .csv files.
```

Comment:
`--s_srs` is pretty terse for a long-form option.  Also, I think [dashes are prefered to underscores](http://stackoverflow.com/questions/1253679/should-command-line-options-in-posix-style-operating-systems-be-underscore-style/1253692#1253692). Perhaps `--srs-source` or `--source-srs`?